CMakeLists.txt

```diff
@@ -46,18 +47,18 @@ endif()
 # We need the NCEPLIBS-sp library.
 find_package(sp 2.3.0 REQUIRED)
 
-# If doxygen documentation we enabled, build it.
-if(ENABLE_DOCS)
-  find_package(Doxygen REQUIRED)
-  set(abs_top_srcdir "${CMAKE_SOURCE_DIR}")
-  add_subdirectory(docs)  
-endif()
-
 # This is the source code directiroy.
 add_subdirectory(src)
 
 # Build tests.
-include(CTest)
 if(BUILD_TESTING)
-    add_subdirectory(tests)
+  include(CTest)
+  add_subdirectory(tests)
+endif()
+
+# If doxygen documentation we enabled, build it.
```

Comment:
@aerorahul was there a reason you moved the docs code down to the bottom of the file?

Reply:
Per CMake best practices;
one first builds the libraries, then the applications, then the tests, and finally the docs.
I was just following that.

Reply:
OK, cool. I am applying these changes to the other NCEPLIBS as well...